changeset.go

```diff
@@ -1323,7 +1368,6 @@ func (cs *Changeset) withUpsertOp(ctx context.Context, tr *ChangesetResource, ol
 		CreationTimestamp: time.Now().UTC(),
 		To:                string(to),
 		Status:            OpStatusCreated,
-		ID:                id,
 	}
 	newUUID, err := uuid.NewRandom()
```

Comment:
If this is randomly chosen, how will `rig revert --uid=<uid>` be used?
Also, if the upsert is for a resource being read from a file, `UID` will initially be empty but `old` is not - hence it will reset it to empty string inside the `if` block won't it?

Reply:
For the first question. `kubectl get changesets -o yaml` to show all operations and their UIDs. Or we can add command like `rig show` to provide the same output.